gslib/commands/cp.py

```diff
@@ -765,26 +764,20 @@
                  cloud than they would if left uncompressed.
 """
 
-_DETAILED_HELP_TEXT = '\n\n'.join([_SYNOPSIS_TEXT,
-                                   _DESCRIPTION_TEXT,
-                                   _NAME_CONSTRUCTION_TEXT,
-                                   _SUBDIRECTORIES_TEXT,
-                                   _COPY_IN_CLOUD_TEXT,
-                                   _CHECKSUM_VALIDATION_TEXT,
-                                   _RETRY_HANDLING_TEXT,
-                                   _RESUMABLE_TRANSFERS_TEXT,
-                                   _STREAMING_TRANSFERS_TEXT,
-                                   _SLICED_OBJECT_DOWNLOADS_TEXT,
-                                   _PARALLEL_COMPOSITE_UPLOADS_TEXT,
-                                   _CHANGING_TEMP_DIRECTORIES_TEXT,
-                                   _COPYING_SPECIAL_FILES_TEXT,
-                                   _OPTIONS_TEXT])
+_DETAILED_HELP_TEXT = '\n\n'.join([
```

Comment:
Boooo. I'm assuming there's a YAPF option to keep long lists split up one-item-per-line like we had before?  The former was much cleaner to read and produced more readable diffs when altering the items.

I think this option is `SPLIT_ALL_COMMA_SEPARATED_VALUES`, and should be changed to True if we want to keep the former style.